tests/upgrades/test_virtwho.py

```diff
@@ -78,16 +77,8 @@ def test_pre_create_virt_who_configuration(
             3. Report is sent to satellite.
             4. Virtual sku can be generated and attached.
         """
-        org = target_sat.api.Organization().search(query={'search': f'name={ORG_DATA["name"]}'})
-        if org:
-            target_sat.api.Organization(id=org[0].id).delete()
-        default_loc_id = (
-            target_sat.api.Location().search(query={'search': f'name="{DEFAULT_LOC}"'})[0].id
-        )
-        default_loc = target_sat.api.Location(id=default_loc_id).read()
         org = target_sat.api.Organization(name=ORG_DATA['name']).create()
-        default_loc.organization.append(target_sat.api.Organization(id=org.id))
-        default_loc.update(['organization'])
+        target_sat.api.Location(organization=[org]).create()
```

Comment:
this location created isn't assigned to any variable, so question is if you really need it in test for pre/post scenarios?

Reply:
> this location created isn't assigned to any variable, so question is if you really need it in test for pre/post scenarios?

The location is assigned to the org "org = target_sat.api.Organization(name=ORG_DATA['name']).create()"

Reply:
yes, it is created with org, but there seems no use of it, right?

Reply:
> yes, it is created with org, but there seems no use of it, right?

@Gauravtalreja1 This location is used for org = target_sat.api.Organization(name=ORG_DATA['name']).create(),  then create a virtwho config in this org

Reply:
Np, it is non-blocking, so going for merge